### PR TITLE
(feat) adds docs template

### PR DESCRIPTION
**Pull request by sjaghori**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,30 +23,33 @@
...
+		"sveltekit-superforms": "^2.14.0",
 		"tailwind-merge": "^2.3.0",
 		"tailwind-variants": "^0.2.1",
 		"typesafe-utils": "^1.16.2",
-		"zod": "^3.23.5"
+		"zod": "^3.23.8"
 	},
 	"devDependencies": {
-		"@eslint/js": "^9.2.0",
+		"@eslint/js": "^9.3.0",
```
Comment (jjnp) [inline package.json:52]: I mean it's alright with me, but why does this PR bump so many versions?

**Review by jjnp — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (sjaghori) [reply package.json:52]: you are right, we shouldn't do that. I had some merge conflicts in the pnpm-lock.yml so I just deleted the file, bumped the versions and reinstalled everything.

**Comment by sjaghori** [comment]
I'm going to merge this, because I only resolved merge conflicts since the last review :)

**Review by benjaminstrasser — APPROVED** [review]
:rocket:

**Review by mledl — APPROVED** [review]
I like that docs 📝

**Pull request merged**